docs/client-sdk/react/tutorials/_category_.json

```diff
@@ -0,0 +1,6 @@
+{
+    "label": "Tutorials for xmtp-react",
```

Comment:
should we say `Tutorials for React SDK` here instead?

Reply:
Beautiful, thank you so much - DONE @rygine!